templates/admin/books/item/change_list.html

```diff
@@ -1,6 +0,0 @@
-{% extends 'admin/change_list.html' %}
-
-{% block object-tools-items %}
-    <li><a href="{% url 'books:items-csv' %}" class="grp-state-focus">Download as CSV</a></li>
```

Comment:
Is it possible to preserve this functionality as a convenience? Is there a way to link to the action to download all items?